app/templates/events/events.html

```diff
@@ -32,52 +48,54 @@
         {{ macros.event(upcoming_events[0]) }}
     </div>
 </section>
-{% if upcoming_events | length > 1 %}
-<section>
-    <div class="content">
-        <div class="triple">
-        {% for idx in range(1,4) %}
-            {% if upcoming_events | length > idx %}
-            {{ macros.small_event(upcoming_events[idx]) }}
-            {% endif %}
-        {% endfor %}
+    {% if upcoming_events | length > 1 %}
+    <section>
+        <div class="content">
+            <div class="triple">
+            {% for idx in range(1,4) %}
```

Comment:
This is for the events triple below the single event.  The first event is at index 0, indexes 1-4 are the three smaller events below the large one.